src/composables/use-focus-on-hide.js

```diff
@@ -1,8 +1,7 @@
-import { watch } from '@vue/composition-api'
-import { isTabbable } from 'reakit-utils/tabbable'
-import { getActiveElement } from 'reakit-utils/getActiveElement'
-import { contains } from 'reakit-utils/contains'
-import { ensureFocus } from 'reakit-utils/ensureFocus'
+import { watch } from '@nuxtjs/composition-api'
```

Comment:
There was an import from `@vue/composition-api` which seems to behave just a little bit differently.